src/browser.ts

```diff
@@ -1,16 +1,10 @@
 import * as protobuf from 'protobufjs';
 import * as gax from './gax';
-
+import {Status} from './status';
+import {OutgoingHttpHeaders} from 'http';
 import {GoogleAuth} from 'google-auth-library';
 import {OperationsClientBuilder} from './operationsClientBrowser'
 
-
-export {
-    CallSettings,
-    constructSettings,
-    RetryOptions,
-  } from './gax';
-
 export {createApiCall} from './createApiCall';
 
 export interface ClientStubOptions {
```

Comment:
can we just re-export `ClientStubOptions` from `grpc.ts`?

`export {ClientStubOptions} from './grpc';`

it's very similar:

```
export interface ClientStubOptions {
  servicePath: string;
  port: number;
  // TODO: use sslCreds?: grpc.ChannelCredentials;
  // tslint:disable-next-line no-any
  sslCreds?: any;
}
```

We might even decide to make use of `servicePath` and `port` later (they are normally used to override the host:port to connect to).